packages/config/index.js

```diff
@@ -130,17 +132,23 @@ BemConfig.prototype.level = function(pathToLevel) {
  * @returns {Promise}
  */
 BemConfig.prototype.library = function(libName) {
-    return this.get().then(function(config) {
-        var libs = config.libs;
-
-        if (!libs) { return; }
-
-        var lib = libs[libName];
-
-        if (!lib) { return; }
-
-        return new BemConfig({ projectRoot: lib.path });
-    });
+    return this.get()
+        .then(function(config) {
+            var libs = config.libs,
+                lib = libs && libs[libName],
+                cwd = lib && lib.path || path.resolve('node_modules', libName);
```

Comment:
`assert(lib === undefined || typeof lib === 'object', 'Invalid `libs` format');` before this line?

To be consistent with sync version